LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.html

```diff
@@ -32,7 +32,7 @@
   // re-use the messageing mechanisms above. In order to not end up in a loop,
   // we'll only click if we don't find fragment in the current URL.
   const target_script = `location.href='${location.href}&continue';`;
-  const target = `javascript:"<script>${target_script}</scri${""}pt>"`;
+  const target = `javascript:${target_script}`;
```

Comment:
https://github.com/web-platform-tests/wpt/pull/44583

WebKit doesn't support JavaScript URLs containing `<script>` tags so this test fails within WebKit regardless of what it should actually be testing. I'm unaware why the test uses this odd construction in the first place. But this removes it.

Reply:
Given https://github.com/web-platform-tests/wpt/pull/44583 has been reviewed, you can merge it there already. Let me know if you need help there.